particles/Layout/Detail.recipes

```diff
@@ -1,11 +1,3 @@
-// @license
-// Copyright (c) 2017 Google Inc. All rights reserved.
-// This code may only be used under the BSD style license found at
-// http://polymer.github.io/LICENSE.txt
-// Code distributed by Google as part of this project is also
-// subject to an additional IP rights grant found at
-// http://polymer.github.io/PATENTS.txt
-
 particle DetailSlider in 'source/DetailSlider.js'
```

Comment:
just confirming - are you omitting the license comment on purpose here and everywhere else, and why?

Reply:
Last I heard these headers are only required on `.js/.css/.html` files.